_dev/src/views/campaign-page.vue

```diff
@@ -85,6 +85,7 @@ export default {
   },
   methods: {
     async getDatas() {
+      this.$store.dispatch('smartShoppingCampaigns/GET_DIMENSIONS_FILTERS', null);
```

Comment:
I think it was fine at the end of the method. The API can work on the Google Ads account at the place, the dimensions can wait for the rest to be loaded.

Reply:
I thought that it was quicker to load the filters if I put it before, but maybe it's just me ?

Reply:
it will, but may impact the time needed to load the more critical details, as all these calls will run in parallel